### DISCUSSION TITLE
EXIF Video Data

### DISCUSSION THREAD

**Discussion by sydjackson17** [olivierfriard/BORIS#592] [Q&A]
I am currently trying to use BORIS for a large amount of videos (over 400 GB's worth). I know BORIS can set a photo observation to the EXIF date and time, but can this be done for multiple videos? I don't see anything on the User Guide about this, and if it's an option on the observations, I can't find it. If it's not possible, would anyone be able to recommend an alternative software so that I can maintain the EXIF data as the observation's date and time? TIA if you can help me.

**Comment by olivierfriard** [comment]
Hi,
I am sorry but I do not understand what do you want to do.

**Reply by sydjackson17** [reply]
Hi,

Apologies for my lack of clarity. Let me re-word my question:

I am attempting to use BORIS for video behavioral analysis. The videos are very short, and I would like to analyze many videos in one observation to save time. It's very important that the date and time from the video be used for this project, but manually setting that for each video will be very time-consuming. However, I don't see an option for extracting the EXIF tag from videos; is that only available for photo observations? Please let me know if further explanation is needed.

**Comment by olivierfriard** [comment]
Hi,
so, you want to code events from your video using the date and time recorded in the EXIF tags?
The timestamp for events should be the sum of this EXIF date/time and the time at which the event occurs in the video.
Can you send me a sample of your video?
